src/components/Codebox/Codebox.scss

```
@@ -23,9 +23,9 @@
     }
 
     button {
-      background-color: var(--brand2);
-      border-radius: 0 0.3rem 0.3rem 0.3rem;
       border-width: 0;
+      border-radius: 0 0.3rem 0.3rem 0.3rem;
+      background-color: var(--brand8);
```

This should fix it I think

```suggestion
      background-color: var(--brand8);
      color: white;
```